packages/taco/test/conditions/context.test.ts

```diff
@@ -55,10 +53,11 @@ describe('context', () => {
     const customParams: Record<string, CustomContextParam> = {};
     customParams[customParamKey] = 1234;
 
-    const contractConditionObj = {
+    const contractConditionObj: ContractConditionProps = {
       ...testContractConditionObj,
       returnValueTest: {
-        ...testReturnValueTest,
+        index: 0,
```

Comment:
`testContractConditionObj` is an ERC20 `balanceOf` contract call that returns a `uint256`, `index` is not applicable.

```suggestion
```